js/phemex.js

```diff
@@ -1833,16 +1838,16 @@ module.exports = class phemex extends Exchange {
             }
         } else if (market['swap']) {
             request['orderQty'] = parseInt (amount);
+            if (stopPrice !== undefined) {
```

Comment:
I think stop orders work for spot markets also, no?

Reply:
https://github.com/phemex/phemex-api-docs/blob/master/Public-Spot-API-en.md#spotPlaceOrder

Reply:
I think, in this case, ByMarkPrice is not working for spot markets, but ByLastPrice will work for spot markets, because spot markets don't have a mark price, but they have a last price. As for contracts – those do have both the last price and the mark price. Let me know if you can place a spot stop order triggered by last price.

Reply:
Nevermind, I misread it a bit. I think it's good to go.